clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -190,9 +189,10 @@ public void testSubscription() {
     @Test(expected = IllegalArgumentException.class)
     public void testSubscriptionOnNullTopicCollection() {
         KafkaConsumer<byte[], byte[]> consumer = newConsumer();
+        List<String> nullList = null;
 
         try {
-            consumer.subscribe(null);
+            consumer.subscribe(nullList);
```

Comment:
This is interesting, so the new method introduces ambiguity in this case. Doesn't seem too bad since `null` is not a valid value though. cc @hachikuji

Reply:
Yeah, seems fine. Either way, the result is an `IllegalArgumentException`.